src/components/__tests__/TechIcon.js

```diff
@@ -20,7 +20,7 @@ describe('Icon Component', () => {
   });
 
   it('contains only one img child tag', () => {
-    const wrapper = mount(<TechIcon {...DEFAULT_PROPS}/>);
+    const wrapper = mount(<TechIcon name="android" style={padding} />);
```

Comment:
there is no actual test with this, 

You would need to create another test like:

it('should add style prop when passed')

it('should override style width and height if size and style props are present')

Reply:
The tests has been added.

Reply:
Leave this with `{...DEFAULT_PROPS}`